### ISSUE TITLE
Stripe Bank not working.

### ISSUE THREAD

**Issue by mahrmediait** [AckerApple/stripe-angular#8]
Hi,

I am tryping to use Stripe-Bank, but it isn't working. There are now inputs shown. Do you have any idea?

Best greetings,

**Comment by AckerApple** [comment]
For stripe-bank you have to construct your own inputs. I'm updating my docs right now for you.

I found it so odd coming to Stripe, from any other payment gateway, that Stripe demands you use their own display. And just when I thought that might be a more secure way to do things, I found the bank portion requires your own inputs.

Look at the stripe-bank part of the demo code and you will see I just put the data into one textarea that becomes serialized before tokenizing:
https://ackerapple.github.io/stripe-angular/


Thoughts? Opinions? Pull Requests? I'll update the github docs now, just slightly

**Comment by mahrmediait** [comment]
Thanks for your answer! Ok, I don't know this, would not be a problem. It is also possible to create a source for a bank account? This isn't working for me, can only create tokens.

Any idea?

Best greetings

**Comment by AckerApple** [comment]
I updated the docs for you in-regards to stripe-bank:
https://github.com/AckerApple/stripe-angular#stripe-bank

On topic of, create a source for a bank account, I tried real hard to get that to work but could not. I thought it would work very similar to sourcing a card but does not appear to. It's like they specifically say the advantages of payment sources is for accepting any/many sources buttttttttttttttt um no I failed to figure out how.

My experience with Stripe has led me to believe the youth built it and it shows. It's savvy, it's pricing is NOT that competitive, and now me having gotten into some depth with it has revealed I'd prefer to be with FirstData again..... Like how savvy cool is it that everything is in cents in Stripe?!?!?

I'm bitter with Stripe. Forced to use it.
